extensions/vertx/runtime/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-virtual-threads</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-kotlin</artifactId>
```

Comment:
That's not great. Now everything depends on Kotlin.

Reply:
It's just a tiny artifact with 3 classes and transitive dependencies on artifacts that are already present, but I agree it is not nice. I took this over from the Scheduler extension without thinking too much about it. I'll ponder over it over the weekend.

Reply:
I think that it's fine. It does not really bring the Kotlin sdk in (it's an optional dependency)...

Reply:
Added one more commit (to be squashed if you guys like it :-) ) that moves the dependency to the `quarkus-kotlin` module. Probably not very nice either, but should be better than a dependency on Kotlin in the `quarkus-vertx` module, I guess.